### DISCUSSION TITLE
Selection of Set

### DISCUSSION THREAD

**Discussion by karsood** [Sunbird-Saral/Community#23] [Ideas]
As of today, there are three selections that are made after logging in: Class, Section, and Subject.

With some states going for repeated examination using different sets (set A, set B, set C, set D), can we have sets come up as a selection after the subject is selected?

Benefit: This will help reduce selection error by the teacher.

If there are no sets for a round of examination in a state, the default entry can be A, as we have it for section selection.

**Comment by dileep-gadiraju** [comment]
We will review this requirement and get back @karsood .

**Comment by dileep-gadiraju** [comment]
@karsood, Please refer  https://github.com/Sunbird-Saral/Community/issues/24. This feature will be available in up coming releases.
